.env.example

```diff
@@ -27,3 +27,9 @@ SEND_GRID_KEY=CANNOT_SEND_REAL_KEY_aFbF_Q-FSTQg.rx0WZ8pllsHBIYwft_liw
 
 # Upload folder
 UPLOADED_FILES_DESTINATION=./uploadedFiles
+
+# AWS S3
+AWS_REGION=us-east-1
+AWS_ACCESS_KEY_ID=AKIATGY45KWDGIOLA4XG
+AWS_SECRET_ACCESS_KEY=AWS_SECRET_ACCESS_KEY
+AWS_PUBLIC_BUCKET_NAME=nestjs-hivein-photo-storage
```

Comment:
why you have real keys in example? example exist because its not secure publish real data to github

Reply:
now you already show the keys, they are stored in git history, its mean that new access should be created. NEVER, NEVER push keys to github